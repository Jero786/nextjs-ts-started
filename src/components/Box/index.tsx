import styled from "styled-components"
import {color} from "styled-system"

const Box = styled.div`
  ${color}
  font-size: 50px;
  background-color: ${({theme}) => theme.colors.blue};
`
export default Box
