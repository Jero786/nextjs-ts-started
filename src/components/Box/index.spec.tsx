import React from "react"

import {render} from "../../utils/tests"
import Box from "."

describe("<Box/>", () => {
  test("Should display text of the box properly", async () => {
    const {asFragment} = render(<Box />)

    const firstRender = asFragment()

    expect(firstRender).toBeDefined()
  })
})
