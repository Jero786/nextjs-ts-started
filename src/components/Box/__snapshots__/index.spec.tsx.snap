// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Box/> Should display text of the box properly 1`] = `
<DocumentFragment>
  <div
    class="Box-sc-13j4q9j-0 hwFXov"
  />
</DocumentFragment>
`;
